src/services/resource.js

```diff
@@ -324,6 +335,12 @@ module.exports = class resourceHelper {
 			res.organization = orgDetails[res.organization_id] ? orgDetails[res.organization_id] : {}
 			res.creator = userDetails[res.user_id] && userDetails[res.user_id].name ? userDetails[res.user_id].name : ''
 			res.notes = res?.meta?.notes ? res.meta.notes : ''
+			if (res?.type == common.RESOURCE_TYPE_PROGRAM && res?.meta?.start_date) {
```

Comment:
@adithyadinesh0412  start date key is need to pass if they have not passed the start_date